pipelines/templates/tasks/validateassets.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  # If true, validation scripts will only be run on the set of changed files
+  # associated with a given pull request. Note that this is only valid to use
+  # for pull request validation, since this functionality requires a pull request
+  # ID in order to determine the two commits to diff. Only relevant if validateCode
+  # is true.
+  scopedValidation: false
+  # The file containing the list of changes, used if scopedValidation is true.  
+  changesFile: ''
+
+steps:
+#v- ${{ if eq(parameters.scopedValidation, false) }}:
```

Comment:
Should this be removed if it's commented out?
Also the comment syntax seems to have an extra `v`

Reply:
it is commented temporarily, pending work being done as part of #8944

Reply:
will fix the extra 'v'

Reply:
fixed the 'v', added comment re: #8944